### PR TITLE
Use summary provided by scanners in email 

### PR DESCRIPTION
**Pull request by dharmit**
Example email:
```
Atomic scanners results for built image registry.centos.org/devcloud/outdated:latest                                                                                         
                                                                                                                                                                             
container-capabilities-scanner:                                                                                                                                              
Dockerfile for the image doesn't have RUN label.                                                                                                                             
Detailed logs link: https://registry.centos.org/pipeline-logs/ZGU1NWRiODdkND/container-capabilities-results.json                                                             
                                                                                                                                                                             
scanner-rpm-verify:                                                                                                                                                          
Libraries and binaries in the image are intact.                                                                                                                              
Detailed logs link: https://registry.centos.org/pipeline-logs/ZGU1NWRiODdkND/RPMVerify_scanner_results.json                                                                  
                                                                                                                                                                             
misc-package-updates:                                                                                                                                                        
Possible updates for packages installed via pip. Possible updates for packages installed via npm. Possible updates for packages installed via gem.                           
Detailed logs link: https://registry.centos.org/pipeline-logs/ZGU1NWRiODdkND/misc_package_updates_scanner_results.json                                                       
                                                                                                                                                                             
pipeline-scanner:                                                                                                                                                            
RPM updates available for the image.                                                                                                                                         
Detailed logs link: https://registry.centos.org/pipeline-logs/ZGU1NWRiODdkND/pipeline_scanner_results.json 
```

### CODE REVIEW

File: container_pipeline/scanners/misc_package_updates.py [hunk 1]
```diff
@@ -59,7 +59,9 @@ def process_output(self, logs):
         """
         data = {}
         data["scanner_name"] = self.scanner_name
-        data["msg"] = "Results for miscellaneous package manager updates"
+        data["msg"] = ""
+        for i in logs:
+            data["msg"] += i["Summary"]
```
Comment (bamachrn) [inline container_pipeline/scanners/misc_package_updates.py]: here do we say we would be having summary for each of the package management we support?

File: container_pipeline/scanners/rpm_verify.py [hunk 2]
```diff
@@ -80,6 +80,6 @@ def process_output(self, json_data):
         data = {}
         data["scanner_name"] = self.scanner_name
         # TODO: More verifcation and validation on the data
-        data["msg"] = "RPM verify results."
+        data["msg"] = json_data["Summary"]
```
Comment (bamachrn) [inline container_pipeline/scanners/rpm_verify.py]: using different style in different scanners? is it by design?

File: container_pipeline/scanners/misc_package_updates.py (hunk 1 above)
Comment (dharmit) [reply container_pipeline/scanners/misc_package_updates.py]: @bamachrn didn't get you. In the email output that I've mentioned in first comment, the line `Possible updates for packages installed via pip. Possible updates for packages installed via npm. Possible updates for packages installed via gem.` that you see is a result of string concatenation of `i["Summary"]` for `pip`, `npm` and `gem` scanner results.

File: container_pipeline/scanners/rpm_verify.py (hunk 2 above)
Comment (dharmit) [reply container_pipeline/scanners/rpm_verify.py]: Different style in what sense? Can you elaborate?

**Comment by navidshaikh** [comment]
is this good to merge ? @dharmit

File: container_pipeline/scanners/misc_package_updates.py (hunk 1 above)
Comment (bamachrn) [reply container_pipeline/scanners/misc_package_updates.py]: got it

File: container_pipeline/scanners/rpm_verify.py (hunk 2 above)
Comment (bamachrn) [reply container_pipeline/scanners/rpm_verify.py]: My thought was if we can have a single type of variable for summary. here we are using string and list. but I see the data['msg'] is having only string. I was not able to get the flow. thanks got it. and looks good to me.

**Comment by navidshaikh** [comment]
Review the PR using github workflow before merge, please.

**Pull request merged**